### ISSUE TITLE
Spell Check Occasionally Marks Words as Spelled Wrong even When Spelled Correctly (Hundreds of Words)

### ISSUE THREAD

**Issue by FeMaster1** [SubtitleEdit/subtitleedit#4436]
I apologize if something like this has already been reported, I attempted to search before posting, but didn't seem to come up with anything similar.

Subtitle Edit 3.5.17, Windows 10 Pro 64-Bit, Version 1909.

This is the second time this has happened to me, the first time I blew it off as I didn't have a clue on how to report it. I was going through a subtitle and doing a spell check on it. The spell checker was flagging hundreds of words as spelled wrong, even though they were correct. Re-typing the word exactly as originally spelled, or selecting the word from the Suggestions box and choosing Use Always made the spell checker happy. There were hundreds of different words throughout the subtitle file that I had to "correct". I'm sure that is has something to do with the specific subtitle file, but I don't know what that would be. I closed and reopened Subtitle Edit and tried again, but the problem remained. I tried another subtitle file afterwards and the problem was not present in that file.

Below are come screen shots of correct words being flagged, and I've attached the subtitle file in question for your inspection and testing.

![Example 1](https://user-images.githubusercontent.com/38301166/96200931-e1682100-0f20-11eb-89cd-28ec5b8eb7f3.jpg)
![Example 2](https://user-images.githubusercontent.com/38301166/96200936-e4631180-0f20-11eb-9f36-d1927e14189b.jpg)
![Example 3](https://user-images.githubusercontent.com/38301166/96200939-e6c56b80-0f20-11eb-837f-37dfc56348c1.jpg)

[Olympus Has Fallen (2013) - Eng.zip](https://github.com/SubtitleEdit/subtitleedit/files/5388607/Olympus.Has.Fallen.2013.-.Eng.zip)

**Comment by niksedk** [comment]
Many letters in the subtitle are special Greek letters.
I've updated "Normalize string" in "Fix common errors" in latest beta to handle these characters (I hope): https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.17/SubtitleEditBeta.zip

**Comment by FeMaster1** [comment]
Thanks again. The Greek Letters would explain why it looked correct, but it didn't recognize the words. I always run the "Fix Common Errors" before a spell check, just to make sure things are as clean as possible before hand. I looked, and just noticed that I did not have "Normalize Strings" enabled, so I have now enabled that. I scanned the file with my current version, but it didn't find anything new, so I will update to the latest beta.

I'm a service technician for office equipment, so I know how frustrating it can be to try and troubleshoot problems without antiquate information. I always try my best to provided as much information as possible to assist in re-creating a problem. It makes things so much easier.

Thanks for all your time and effort put into this!
